catalog/spatial/geocoding/spatial-geocoding-create/src/test/java/org/codice/ddf/spatial/geocoding/create/TestGeoNamesCreator.java

```diff
@@ -33,7 +33,8 @@ public void testNoEmptyFields() {
                 -111.84125,
                 "PPL",
                 236123,
-                "Candler,Candleris,Chandler,Chandlur");
+                "Candler,Candleris,Chandler,Chandlur",
+                "US");
```

Comment:
Sounds like we decided to use 3 letter country codes here, any progress on that?